src/Swarm/Language/LSP/Hover.hs

```diff
@@ -176,17 +176,6 @@ explain len mMod trm = case trm of
 --
 -- Note that 'Force' is often inserted internally, so
 -- if it shows up here we drop it.
---
--- >>> tprint = error . ("This is fine:\n" <>) . T.unpack
```

Comment:
On second thought, I could have dropped this hack instead and figured out what the problem is.